code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -245,7 +245,7 @@
 
 /obj/item/gun/ballistic/shotgun/doublebarrel/hook
 	name = "hook modified sawn-off shotgun"
-	desc = "Range isn't an issue when you can bring your victim to you."
+	desc = "Range isn't an issue when you can bring your victims close. The only thing they fear is you." //Funny DOOM reference
```

Comment:
You know, I really did make this before the DOOM with the hook shotgun came out and I'm a bit cheesed that every AAA game is getting a hook now but whatever I didn't put any fluff here so this is fine

Reply:
that means Doom devs play ss13 and copied your idea